hs_file_types/models/base.py

```diff
@@ -898,6 +964,16 @@ def is_fileset(self):
         """Return True if this aggregation is a fileset aggregation, otherwise False"""
         return self.get_aggregation_class_name() == 'FileSetLogicalFile'
 
+    @property
+    def is_model_program(self):
```

Comment:
From the code that I've seen so far, you don't need to know whether a logical file is_model_program, etc... You really want to know whether a logical file is file or folder based.